test/test-parse-statement.cpp

```diff
@@ -717,6 +717,25 @@ TEST_F(Test_Parse_Statement, else_without_if) {
                               "visit_exit_block_scope",
                           }));
   }
+
+  {
```

Comment:
This test case is inside Test_Parse_Statement.else_without_if, but this test case is an `else` *with* and `if`. I think you should make a separate `TEST_F`. Perhaps `Test_Parse_Statement.if_else_with_malformed_condition`? (Naming is hard.)

Reply:
Thanks for catching that. I moved the new test into a separate `TEST_F` with the name you suggest.